docs/source/class_reference/Structure.rst

```diff
@@ -0,0 +1,156 @@
+    
+        + the chain, residue division of atoms (inter-residue connectivity)
+    
+          & the canonical residue name and atom names in it (intra-residue connectivity)
+    
+          & the connectivity table for non-canonical parts
+    
+        + the connectivity table for the whole enzyme
+    
+    - coordinate (atomic coordinate upon topology)
+    
+    Base on this concept, Structure object holds a list of composing Chain objects which also contain Residue objects and so that Atom objects. In each Atom     object, atom name and coordinate is stored. Every level (chain, residue, atom/coordinate/connectivity) of information can all be pulled out from the Structure     object with getter methods and can be set with setter methods. Also Structure() supports common editing methods such as add/remove children objects.
+    
+    
+    Application
+    ------------------------------------------------
+    
+    Application of Structure objects - Binding modules:
+    
+    Generation:
+    
+        Note that Structure() objects SHOULD NOT be created by the user directly and instead created through different generation methods from the binding     StructureIO classes (e.g.: ``enzy_htp.structure_io.PDBParser().get_structure()``) from different file types and different external data structures.
+    
+    Selection:
+    
+        Selection of Structural regions are handled by the Selection module.
+    
+    Operation:
+    
+        Changes of the Structure data are handled by functions in the operation module. These commonly used operations of Structure will than be used in     scientific APIs: Preparation, Mutation, Geom Variation. And structure based descriptors are derived by functions in the Energy Engine module.
+    
+    Details
+    ------------------------------------------------
+    
+        Sturcture() is designed for direct interfacing by users.
+    
+        Composed of child Chain() objects and their subsequent child Residue() objects and so on Atom() objects.
+    
+        Note: This class SHOULD NOT be created directly by users. It should be created with methods from the StructureIO module.
+    
+        Note: regardless of index assigned to residues and atoms, there is an intrinsic indexing system based on the order of _children lists. This intrinsicc index can be compared with pymol's index (not id)
+
+Highlighted Methods ``get``
+==============================================
+
+The ``get`` method is designed to retrieve specific components—either a Chain, a Residue, or an Atom—from a biological structure based on a given key. The key must follow a specific format, and the method performs differently based on the level of detail in the key.
+
+Input
+---------------------------------------------------------
+``key``: A string that specifies the chain, residue, and atom in a biological structure using the format ``<chain_name>.<residue_index>.<atom_name>``. The levels of detail can vary:

```

Comment:
Let's format this to be the same as other pages. See QM Single Point for example

Reply:
i.e., the name of the argument will have its own line